cmd/flags.go

```diff
@@ -148,7 +148,11 @@ func clientFlags() []cli.Flag {
 			Value: time.Hour,
 			Usage: "interval to automatically backup metadata in the object storage (0 means disable backup)",
 		},
-
+		&cli.DurationFlag{
+			Name:  "heartbeat",
+			Value: time.Minute,
```

Comment:
Should we change it to 10 seconds? There will be 10 qps for 1000 clients, kind of reasonable.

Reply:
sure